src/importkeydialog.cpp

```
@@ -0,0 +1,141 @@
+  if (importFromString(input)) {
+    accept();
+  }
+}
+
+void ImportKeyDialog::on_inputTextEdit_textChanged() {
+  const bool hasInput = !ui->inputTextEdit->toPlainText().trimmed().isEmpty();
+  ui->importButton->setEnabled(hasInput);
+  ui->buttonBox->button(QDialogButtonBox::Ok)->setEnabled(hasInput);
+}
+
+bool ImportKeyDialog::importFromString(const QString &input) {
```

Clang (14.0.6)

[error] use a trailing return type for this function

(modernize-use-trailing-return-type,-warnings-as-errors)